src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar search-bar--bg">
+      <form
+      action="#"
+      method="get"
+      class="search-bar__form"
+      data-qa="big"
+      >
+        <label for="search-bar__input">
+          <img
+          class="search-bar__icon search-bar__icon--bg"
+          src="images/Search.svg"
+          alt="search-icon"
+          >
+        </label>
+        <input
+          id="search-bar__input"
+          type="text"
+          class="search-bar__input search-bar__input--bg"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </form>
+    </div>
+    <div class="search-bar search-bar--sm">
+      <form
+      action="#"
+      method="get"
+      class="search-bar__form search-bar__form--sm"
+      data-qa="small"
+      >
+        <label for="search-bar--sm__input">
+          <img
+          class="search-bar__icon search-bar__icon--sm"
```

Comment:
read the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 2 - there shouldn't be `--bg`  and `--sm` only common and one of the listed, write styles correspondingly